dt/uni04delta/edpm-pre-ceph/deployment/kustomization.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  # Set namespace to OpenStack on all namespaced objects without a namespace
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+components:
+  - ../../../../lib/dataplane/deployment
+
+replacements:
+  - source:
+      kind: ConfigMap
+      name: edpm-values
```

Comment:
This should be called `edpm-deployment-values` as defined in `examples/dt/uni04delta/edpm-pre-ceph/values.yaml`